apps/svelte.dev/src/routes/banner.json/+server.js

```diff
@@ -9,29 +9,15 @@ export const GET = async () => {
 	return json(
 		defineBanner([
 			{
-				id: 'advent2023',
-				start: new Date('1 Dec, 2023 00:00:00 UTC'),
-				end: new Date('24 Dec, 2023 23:59:59 UTC'),
+				id: 'sveltehack2024',
+				start: new Date('22 Oct, 2024 00:00:00 UTC'),
+				end: new Date('10 January, 2025 23:59:59 UTC'),
 				arrow: true,
 				content: {
-					lg: 'Advent of Svelte 2023 is here!',
-					sm: 'Advent of Svelte'
+					lg: 'Svelte Hack 2024 (Runes edition) is here!',
```

Comment:
```suggestion
					lg: 'Celebrating the release of Svelte 5 with Svelte Hack 2024!',
```

Reply:
This reads oddly. Who is celebrating? It should probably be either "We’re celebrating the release" or "Celebrate the release" if we're going with this language. It should probably be more CTA-ish anyway though — something like this?

> Cast runes, win prizes: SvelteHack 2024

Most usages on the site are 'SvelteHack' (though the `<title>` is 'Svelte Hack' and there's at least one 'Sveltehack'). We need to ensure consistency.

Reply:
I wanted to put "Svelte 5" in in case someone didn't realize that already. Though I guess it becomes obvious once you click the link ... yeah I think I like your suggestion.